tests/src/Unit/MergeYamlTest.php

```diff
@@ -2,19 +2,118 @@
 
 namespace EdisonLabs\MergeYaml\Unit;
 
+use EdisonLabs\MergeYaml\MergeYaml;
 use PHPUnit\Framework\TestCase;
+use Symfony\Component\Yaml\Yaml;
 
 /**
  * Tests generation of merge-yaml.
  */
 class MergeYamlTest extends TestCase
 {
+    /**
+   * The directory where the merged files will be placed.
+   *
+   * @var string
+   */
+    public $outputDir;
+
+    /**
+     * The file name patterns to scan for.
+     *
+     * @var array
+     */
+    public $fileNamePatterns;
+
+    /**
+     * The paths to scan recursively for yaml files.
+     *
+     * @var array
+     */
+    public $sourcePaths;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        $this->fileNamePatterns = [
+          'test',
+        ];
+        $this->sourcePaths = [
+            dirname(__FILE__).'/../../assets',
+        ];
+        $this->outputDir = dirname(__FILE__).'/../../assets';
```

Comment:
The output directory is the same as the source directory. I think it should be different, like `/tmp/merge-yaml`